platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java

```diff
@@ -128,8 +128,8 @@ protected DocumentBuilder initialValue() {
 
     private static SAMLObjectBuilder<Issuer> issuerBuilder;
 
-    private static XMLObjectBuilderFactory builderFactory =
-            XMLObjectProviderRegistrySupport.getBuilderFactory();
```

Comment:
Might want to check your formatter. I'm not getting the same results as you when formatting this file.